src/main/webapp/app/shared/markdown-editor/commands/orderedListCommand.ts

```diff
@@ -11,24 +11,23 @@ export class OrderedListCommand extends Command {
      * @desc Use the Markdown language for creating/removing an ordered list
      */
     execute(): void {
-        const selectedText = this.getSelectedText();
-        this.splitText(selectedText);
+        const extendedText = this.getExtendedSelectedText();
+        this.splitText(extendedText);
     }
 
     /**
      * Splits the text and performs the necessary manipulations.
      * @function splitText
      * @param selectedText the selected text by the cursor
      */
-    splitText(selectedText: string): void {
-        const parseArray = selectedText.split('\n');
+    splitText(selectedText: string[]): void {
```

Comment:
This method should be renamed, as it no longer splits the text, but only triggers the manipulation for each line.

The same applies to the method with the same name in the unordered list.